### ISSUE TITLE
Support for SQL server as backend

### ISSUE THREAD

**Issue by spanhotra** [kestra-io/kestra#3810]
### Feature description

Kestra has support for MySQL and Postgres.
We are a MSSql shop,
Is there a plan for MSSql backend?
Or if there is a way we can use that?

**Comment by anna-geller** [comment]
You're right that, currently, Kestra open-source supports MySQL and Postgres. We understand the importance of MS SQL for your organization. While there are no immediate plans to include MS SQL support in the open-source edition, we could consider this feature for our Enterprise Edition. Let me know if you would like to discuss this further.

**Comment by spanhotra** [comment]
> You're right that, currently, Kestra open-source supports MySQL and Postgres. We understand the importance of MS SQL for your organization. While there are no immediate plans to include MS SQL support in the open-source edition, we could consider this feature for our Enterprise Edition. Let me know if you would like to discuss this further.

Thanks for the response. 
Let's discuss this.
